hw/top_earlgrey/ip/alert_handler/data/autogen/alert_handler.hjson

```diff
@@ -56,21 +56,80 @@
     { name: "NLpg",
       desc: "Number of LPGs.",
       type: "int",
-      default: "1",
+      default: "20",
       local: "true"
     },
     { name: "NLpgWidth",
       desc: "Width of LPG ID.",
       type: "int",
-      default: "1",
+      default: "5",
       local: "true"
     },
-    { name: "AlertLpgMap",
+    { name: "LpgMap",
       desc: '''
             Defines a mapping from alerts to LPGs.
             '''
       type: "logic [NAlerts-1:0][NLpgWidth-1:0]",
-      default: "'0",
+      default: '''
+{5'd19,
```

Comment:
same spacing nit

Reply:
jep, this was a templating issue and is fixed now.